_posts/05-06-01-Internationalization-and-Localization.md

```diff
@@ -12,27 +12,27 @@ words - in our case, internationalization becomes i18n and localization, l10n._
 First of all, we need to define those two similar concepts and other related things:
 
 - **Internationalization** is when you organize your code so it can be adapted to different languages or regions
-without refactorings. This is usually done once - preferably, in the beginning of the project, or else you'll probably
-need some huge changes in the source!
+without refactorings. This action is usually done once - preferably, at the beginning of the project, or else you will
+probably need some huge changes in the source!
 - **Localization** happens when you adapt the interface (mainly) by translating contents, based on the i18n work done
 before. It usually is done every time a new language or region needs support and is updated when new interface pieces
 are added, as they need to be available in all supported languages.
-- **Pluralization** defines the rules needed between different languages to interoperate strings containing numbers and 
-counters. For instance, in English when you have only one item, it's singular, and anything different from that is 
+- **Pluralization** defines the rules required between distinct languages to interoperate strings containing numbers and 
+counters. For instance, in English when you have only one item, it is singular, and anything different from that is 
 called plural; plural in this language is indicated by adding an S after some words, and sometimes changes parts of it.
 In other languages, such as Russian or Serbian, there are two plural forms in addition to the singular - you may even
 find languages with a total of four, five or six forms, such as Slovenian, Irish or Arabic.
 
 ## Common ways to implement
 The easiest way to internationalize PHP software is by using array files and using those strings in templates, such as
-`<h1><?=$TRANS['title_about_page']?></h1>`. This is, however, hardly a recommended way for serious projects, as it poses
+`<h1><?=$TRANS['title_about_page']?></h1>`. This way is, however, hardly recommended for serious projects, as it poses
```

Comment:
This reads awkward.

Suggest changing `This way is, however, hardly recommended` to: `However, this is not recommended` (Or if Grammarly still wants "this way" instead of just "this", I guess that's okay even if it's more strict.